### PR TITLE
Support plugin installation outside of Wayfire’s prefix

### PR DESCRIPTION
**Pull request by alyssais**
Part 2 of #493. Implemented as proposed there. Also changes autostart to start wf-panel and wf-background from PATH rather than Wayfire’s prefix.

### CODE REVIEW

File: src/output/plugin-loader.cpp [hunk 1]
```diff
@@ -147,16 +148,38 @@ void plugin_manager::reload_dynamic_plugins()
...
     while(stream >> plugin_name)
     {
         if (plugin_name.size())
         {
             if (plugin_name.at(0) == '/')
+            {
                 next_plugins.push_back(plugin_name);
+                continue;
+            }
             else
```
Comment (ammen99) [inline src/output/plugin-loader.cpp:171]: I imagine you don't want to have this `else` block anymore ...

Comment (ammen99) [reply src/output/plugin-loader.cpp:171]: ping ?

Comment (alyssais) [reply src/output/plugin-loader.cpp:171]: Sorry for being slow.

> I imagine you don't want to have this `else` block anymore ...

Probably not here. We should either drop it, or add the library name as a fallback after the loop through plugin_prefixes. The latter would have the effect of following back to the standard library search paths. What do you think?

Comment (ammen99) [reply src/output/plugin-loader.cpp:171]: I think simply removing is fine. Searching the `/usr/lib` or whatever doesn't seem a good idea, and the code doesn't really do it - if you look closely, it searches in the default plugin path which is checked below anyway.

**Comment by ammen99** [comment]
Thanks a lot!

**Pull request merged**